logging/processors.rst

```diff
@@ -184,6 +189,8 @@ Symfony's MonologBridge provides processors that can be registered inside your a
     The ``RouteProcessor`` and the ``ConsoleCommandProcessor`` were introduced
     in Symfony 4.3.
 
+Note that Monolog have also some built in processors ready to use.
```

Comment:
Hmm. This IS a nice thing to know, especially if there are some useful ones. But, how can I find them? If there an easy place to link to to show what processors are available, a link would be awesome.

Reply:
Ok i will update and point to the github processors part of the repo of monolog
Should i point a specific version or master is ok ?